src/test/java/org/example/TasksLifeCycleTest.java

```diff
@@ -27,6 +27,9 @@ public void smokeToDoMVC () {
         toggle("2");
         assertItemsLeft(3);
 
+        refresh();
+        sleep(1000);
```

Comment:
Could you remove sleeps in your tests?

Reply:
No, because of the app's architecture. When it's fixed, I'll do that.